src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -147,11 +147,11 @@ def init_server_param(self):
         self.hostlist_servers = self.params.get("test_machines", '/run/hosts/*')
         self.server_group = self.params.get("server_group", '/server/',
                                             'daos_server')
+        self.nvme_parameter = self.params.get("bdev_class", '/server_config/',
+                                              None)
 
     def spdk_setup(self):
```

Comment:
Now that this is all being done in _apricot_, I don't know that there is a lot of value in this method any more.  It can probably just be collapsed down into the `setUp()` method directly.

Reply:
Agreed - same goes for init_server_param(), as mentioned below.  Also this appears to be an old version of test.py